portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Joey's Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Image Gallery</h1>
+      <p>Click <a href="index.html">here</a> to see return home.</p>
+      <p>Here are some images of my life at the moment:</p>
+      <p></p>
```

Comment:
I don't write much html, but I think using empty paragraph tags for formatting is not recommended. See https://stackoverflow.com/questions/14848326/empty-paragraph-tags-should-i-allow-in-html-editor-or-not

Reply:
I removed the <p></p> and put <br> inside the paragraph tags to format correctly